worker/reference/api/index.md

```diff
@@ -703,25 +703,26 @@ Tasks have timeouts associated with them that specify the amount of time (in sec
 #### Endpoint
 
 <div class="grey-box">
-GET /projects/<span class="variable project_id">{Project ID}</span>/tasks?code_name={CODE NAME}
+GET /projects/<span class="variable project_id">{Project ID}</span>/tasks
 </div>
 
 #### URL Parameters
 
 * **Project ID**: The ID of the project whose tasks you want to get a list of.
 
-#### Required Query Parameters
-
-* **code_name**: The name of your worker (code package).
```

Comment:
I think the `code_name` parameter is not required because this query can be executed successfully and return all tasks without this param